x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -83,7 +83,7 @@ export class Simulator {
 
     // If needed, create a fake 'history' instance.
     // Resolver will use to read and write query string values.
-    this.history = history ?? createMemoryHistory();
+    this.history = history ?? createBrowserHistory();
```

Comment:
Oops. dont need this. originally I thought I would use the simulator in the test plugin.